package.json

```diff
@@ -16,18 +16,54 @@
       "transform-flow-strip-types"
     ]
   },
+  "eslintConfig": {
+    "extends": "ello",
+    "globals": {
+      "AndroidInterface": false,
+      "ENV": false
+    }
+  },
+  "stylelint": {
+    "extends": "stylelint-config-primer",
+    "rules": {
+      "comment-empty-line-before": null,
+      "selector-class-pattern": null,
+      "selector-no-type": null,
+      "no-unsupported-browser-features": [
+        true,
+        {
+          "browsers": "> 5%, last 2 firefox versions, last 2 chrome versions, last 2 safari versions, last 2 edge versions, ie 11, not OperaMini all",
+          "ignore": [
+            "calc",
+            "flexbox",
+            "viewport-units",
+            "will-change"
+          ]
+        }
+      ]
+    }
+  },
```

Comment:
We don't need any of the style lint stuff in here. This repo shouldn't ever have CSS.